src/components/views/messages/TextualBody.js

```diff
@@ -110,16 +113,12 @@ export default class TextualBody extends React.Component {
             // Highlight code
             const codes = ReactDOM.findDOMNode(this).getElementsByTagName("code");
             if (codes.length > 0) {
-                for (let i = 0; i < codes.length; i++) {
-                    // Do this asynchronously: parsing code takes time and we don't
```

Comment:
Let's keep this comment, it seems like a helpful reminder to future readers on why this `setTimeout` is here.